solr/solr-ref-guide/src/solrj.adoc

```diff
@@ -65,16 +65,16 @@ To compile code manually that uses SolrJ, use a `javac` command similar to:
 
 [source,bash,subs="attributes"]
 ----
-javac -cp .:$SOLR_HOME/dist/solr-solrj-{solr-docs-version}.0.jar ...
+javac -cp .:$SOLR_HOME/server/solr-webapp/webapp/WEB-INF/lib/solr-solrj-{solr-docs-version}.0.jar ...
 ----
 
 At runtime, you need a few of SolrJ's dependencies, in addition to SolrJ itself.
-For convenience, these dependencies are made available in the `dist/solrj-lib` directory.
+In the Solr distribution these dependencies are not separated from Solr's dependencies, so you must include all.
```

Comment:
Rather than ask users to add ALL solr's jars, perhaps link to e.g. https://search.maven.org/artifact/org.apache.solr/solr-solrj/8.11.1/jar and tell them that just a few of those are required, and depending on their use case they may include zookeeper jars and netty jars (zk https).

Reply:
How about:

In the Solr distribution these dependencies are not separated from Solr's dependencies, so you must include all or manually choose the exact set that is needed. Please refer to the maven release for the exact dependencies needed for your version: https://search.maven.org/artifact/org.apache.solr/solr-solrj/<solr-version>/jar